recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "pydantic" %}
-{% set version = "1.9.0" %}
+{% set version = "1.10.2" %}
 {% set repo_url = "https://github.com/samuelcolvin/pydantic" %}
```

Comment:
The repo has changed to https://github.com/pydantic/pydantic

Reply:
Sorry, I thought I saved that change when updating the urls. Thank you!